components/cluster/command/check.go

```diff
@@ -414,8 +414,7 @@ func fixFailedChecks(ctx *task.Context, host string, res *operator.CheckResult,
 		operator.CheckNameMem,
 		operator.CheckNameCommand,
 		operator.CheckNameFio:
-		// don't show unsupported message for checks that are impossible to fix by us
-		return "", nil
+		msg = fmt.Sprintf("%s, auto fixing not possible", res)
```

Comment:
This case section can be just deleted in whole, use the `default` is enough.

Reply:
Comment addressed. PTAL.